ci/templates/ci/user_management_detail.html

```diff
@@ -0,0 +1,195 @@
+              {% endwith %}
+            </tr>
+            <tr align='center'>
+          <th>Unique ID:</th>
+          {% url 'identities-detail' identity.id as url %}
+            <td><a href="{{ url }}">{{ identity.id|truncatechars:12 }}</a></td>
+        </tr>       
+        <tr align='center'>
+          <th>Receiver role</th>
+         <td> {{ identity.details.receiver_role }}</td>
+        </tr>
+        <tr align='center'>
+            <th>Registration date</th>
+            <td>{{ identity.created_at|get_date|date:"D d M Y H:i" }} </td>
+         </tr>
+         <tr align='center'>
+                    <th>Linked to:</th>
+                    {% if identity.details.linked_to == None %}
+                    <td>Mother is not linked to a gatekeeper </td>
+                      {% else %}  
+                      {% url 'identities-detail' identity.details.linked_to as url %}                 
+          <td> <a href="{{ url }}"> {{ identity.details.linked_to }}</a></td>
+          {% endif %}
+        </tr>
+        <tr align='center'>
+            <th>Linked role:</th>
+            <td>{{ linked_to.details.receiver_role|default_if_none:""|title  }}</td>         
+        </tr>
+        </tbody>
+      </table>   
+      </div>
+    </div>
+      
+
+    <div class="table-full">
+        <div class="table-responsive">
+          <table class="table" data-sort="table">
+              <tbody>
+              {% for registration in registrations.results %}
+              <tr align='left'>
+                  <th>Gravida:</th>               
+              <td>{{ registration.data.gravida |default_if_none:""|title }}</td>
+            </tr>            
+            <tr align='left'>
+                <th>Validated registration:</th> 
+                {% url 'registrations-detail' registration.id as url %}
+                <td><a href="{{ url }}">{{ registration.validated }}</a></td>                        
+          </tr>
+          <tr align='left'>
+              <th>Invalid fields:</th>               
+          <td>{{ registration.data.invalid_fields |default_if_none:""|title }}</td>
+        </tr>            
+            {% endfor %}
+          </tbody>
+        </table>
+      </div>
+    </div>
+
+
+      <h3>Message Details</h3>
+      <div class="table-full">
+          <div class="table-responsive">
+            <table class="table" data-sort="table">
+                {% for subscription in subscriptions.results|dictsortreversed:'active' %}
+              <tbody>
+                <tr align='center'>                           
+                {% if subscription.active == True %}
+                <th>Current Message Set</th>   
+                <td>{{ messagesets|get_item:subscription.messageset }}</td>                                
+                   {% elif subscription.active == False %}
+                   <th>Current Message Set</th> <td>No active subscription</td>
+                   {% else %}
```

Comment:
Thanks for the feedback Erik. The else was for a situation where the user has no subscription information. I've seen some identities without a subscription so I thought it might be useful to say something in such cases. Not necessary? Also since an identity can have multiple subscriptions, wouldn't I need to loop through first to check which ones are active/inactive (so an IF inside a FOR)?

Reply:
If the else is in the loop but there is not subscriptions to loop it will never get there, thats why I said move it outside the loop.
So:
`if there is subscriptions: do the loop`
`else: display message that there is no subscriptions`